ionic2/app/components/poke-filter-time-tab/poke-filter-time-tab.component.ts

```diff
@@ -30,8 +30,8 @@ export class PokeFilterTimeTabComponent implements OnInit {
     setTimeout(() => this.cdr.detectChanges(), 100);
   }
 
-  onSightingsRangeChanged(event) {
-    this.filter.sightingsRange = event.value;
+  onSightingsRangeChanged(value) {
```

Comment:
Usually, when working with Form elements, Angular2 passes the whole event. Since this is an ionic Event, only the new Value is passed in the arguments. To make this clear, I changed the naming.